reference/component-versions.md

```diff
@@ -5,6 +5,67 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, they are marked with `n/a` below.
 
+## 1.23.0
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | ? |
+| `ams-node-controller` | `1.23/stable` | ? |
```

Comment:
Should we call out the ams-node-controller as deprecated here as well? For both charms and snaps?